modelci/types/vo/model_vo.py

```diff
@@ -260,32 +261,3 @@ def from_bo(model_bo: ModelBO):
             creator=model_bo.creator,
             create_time=model_bo.create_time,
         )
-
-
-class ModelIn(BaseModel):
```

Comment:
`ModelIn` import  should be removed from `modelci/types/vo/__init__.py `

Reply:
Fixed